packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -700,4 +700,28 @@
     <label>Rechercher à la place {{0}}</label>
     <name>quantic_SearchInsteadFor</name>
   </customLabels>
+  <customLabels>
+    <label>Non pertinent</label>
+    <name>quantic_Irrelevant</name>
+  </customLabels>
+  <customLabels>
+    <label>pas précis</label>
+    <name>quantic_NotAccurate</name>
+  </customLabels>
+  <customLabels>
+    <label>Pas à jour</label>
+    <name>quantic_OutOfDate</name>
+  </customLabels>
+  <customLabels>
+    <label>Nocif</label>
+    <name>quantic_Harmful</name>
+  </customLabels>
+  <customLabels>
+    <label>Commentaires</label>
+    <name>quantic_Feedback</name>
+  </customLabels>
+  <customLabels>
+    <label>Pourquoi la réponse générée n'a-t-elle pas été utile ?</label>
```

Comment:
@mmitiche @SimonMilord 
In RGA, users could select more than 1 option (when I wrote the copy, could've changed...?). 

But if only 1 answer is possible, here's the French: 
Sélectionnez la raison qui explique pourquoi la réponse générée n'était pas utile.

Reply:
Yep this has been changed, now users will be able to select only one option.